app/src/scenes/Craving/DefineStrategy.js

```diff
@@ -30,22 +30,35 @@ const DefineStrategy = ({ navigation, route }) => {
   const { strategyIndex } = route.params;
   const [strategies, setStrategies] = useRecoilState(defineStrategyState);
   const setCurrentStrategyIndex = useSetRecoilState(currentStrategyState);
-  const strategy = strategies.find((strategy) => strategy.index === strategyIndex) ?? {
+  const toast = useToast();
+
+  const defaultStrategy = {
     index: strategyIndex,
     intensity: 5,
+    feelings: [],
+    actionPlan: [],
+    trigger: [],
   };
-  const [feelings, setFeelings] = useState(strategy.feelings ?? []);
-  const [actionPlan, setActionPlan] = useState(strategy.actionPlan ?? []);
-  const [trigger, setTrigger] = useState(strategy.trigger ?? []);
-  const [selectedIntensity, setSelectedIntensity] = useState(strategy.intensity);
-  const toast = useToast();
+
+  useEffect(() => {
```

Comment:
non ça ne marche pas comme ça

Reply:
https://react.dev/learn/you-might-not-need-an-effect il faut que tu relises ce passage, et tu vas trouver

Reply:
(on peut voir ensemble bien sûr)